checker.go

```diff
@@ -526,13 +449,13 @@ func (c *stringContainsChecker[T]) Args() []Arg {
 
 // SliceContains returns a Checker that succeeds if the given
 // slice contains the given element, by comparing for equality.
-func SliceContains[T comparable](container []T, elem T) Checker {
+func SliceContains[T any](container []T, elem T) Checker {
 	return SliceAny(container, F2(Equals[T], elem))
 }
 
 // MapContains returns a Checker that succeeds if the given
 // map contains the given value, by comparing for equality.
-func MapContains[K, V comparable](container map[K]V, elem V) Checker {
+func MapContains[K comparable, V any](container map[K]V, elem V) Checker {
```

Comment:
worth moving this to a separate commit with a test? or is one of the runnable examples already covering this?

Reply:
good idea. Reverted for now.